metal3-dev/local-bmo.sh

```diff
@@ -67,8 +67,13 @@ oc apply -f $OUTDIR/bmo-deployment-dev.yaml -n openshift-machine-api
 # Set some variables the operator expects to have in order to work
 export OPERATOR_NAME=baremetal-operator
 
+CLUSTER_IP=${CLUSTER_PROVISIONING_IP}
+if [[ "${IP_STACK}" == "v6" ]]; then 
+  CLUSTER_IP="["${CLUSTER_PROVISIONING_IP}"]"
```

Comment:
utils.sh has a function you can use directly, see wrap_if_ipv6

Reply:
Nice one, thanks!